java/sage/media/sub/VobSubSubtitleHandler.java

```diff
@@ -46,24 +54,25 @@ public void loadSubtitlesFromFiles(sage.MediaFile sourceFile)
     try
     {
       subFile = getLoadableSubtitleFile(sourceFile, new java.io.File(subs[0].getPath()));
-      inStream = sage.IOUtils.openReaderDetectCharset(subFile, sage.Sage.BYTE_CHARSET);
+      inStream = sage.IOUtils.openReaderDetectCharset(subFile, sage.Sage.BYTE_CHARSET, sourceFile.isLocalFile());
```

Comment:
I spotted that too. It's crazy. You'd think I didn't look this over a few times before I pushed it.